### PR TITLE
 Admin Panel - Analytics #2952 

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: libs/admin/src/lib/admin-panel/pages/event/event.component.html [hunk 1]
```diff
@@ -1,19 +1,41 @@
-<h1><a routerLink="/c/o/admin/panel/">Administration panel</a> > Event > {{ eventId }} </h1>
-<section fxLayout="row" fxLayoutAlign="space-around start" *ngIf="event">
+<section fxLayout="column">
+  <h1><a routerLink="/c/o/admin/panel/">Administration panel</a> > Event > {{ eventId }} </h1>
+  <article fxLayout="row" fxLayoutAlign="space-around start" *ngIf="event">
 
     <admin-event-info [event]="event"></admin-event-info>
+    <section>
```
Comment (Mussuranae) [inline libs/admin/src/lib/admin-panel/pages/event/event.component.html:6]: Here also for the `section`, you can put `main` first, `section` in secondary and `article` after `section`, but it doesn't look great to put `section` in `article`.

File: libs/admin/src/lib/admin-panel/pages/event/event.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+section {
+  margin: 40px;
+}
+
+mat-card {
+  margin-bottom: 40px;
+}
+
+.poster {
+  height: 50px;
```
Comment (Mussuranae) [inline libs/admin/src/lib/admin-panel/pages/event/event.component.scss:10]: Same, poster aren't square, put a little more height.

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.html [hunk 3]
```diff
@@ -1,191 +1,193 @@
...
+            <mat-card-title>Contact Information</mat-card-title>
+          </mat-card-header>
+          <mat-card-content fxLayout="column">
+            <article formGroupName="denomination">
+              <mat-form-field appearance="outline" class="full-width">
+                <mat-label>Company Name</mat-label>
+                <input test-id="name" matInput placeholder="Company Name" formControlName="full" type="text" (blur)="uniqueOrgName()" required/>
+              </mat-form-field>
+              <mat-error *ngIf="orgForm.get('denomination').hasError('required')">Company name is required</mat-error>
+              <mat-error *ngIf="orgForm.get('denomination').get('full').hasError('notUnique')">The name is already taken</mat-error>
```
Comment (Mussuranae) [inline libs/admin/src/lib/admin-panel/pages/organization/organization.component.html:31]: Put these two `mat-error` in the `mat-form-field`, there will be a large space between them if you put error outside the form-field.

File: libs/admin/src/lib/admin-panel/pages/organization/organization.component.html [hunk 4]
```diff
@@ -1,191 +1,193 @@
...
-    <admin-image-sizes [imgRef]="org.logo">
-      <h3>Image sizes</h3>
-    </admin-image-sizes>
-  </article>
-  <article *ngIf="orgForm" fxLayout="column" class="main-article">
-    <form [formGroup]="orgForm" fxLayout="column">
-
-       <!-- Contact Information -->
-      <mat-card>
+<section fxLayout="column">
```
Comment (Mussuranae) [inline libs/admin/src/lib/admin-panel/pages/organization/organization.component.html:1]: This very first `section` can be a `main` tag.

File: libs/admin/src/lib/admin/toolbar-top/toolbar-top.component.html [hunk 5]
```diff
@@ -1,8 +1,45 @@
 <mat-toolbar fxLayout="row" fxLayoutAlign="space-between center" id="admin-toolbar">
 
-  <button mat-button routerLink="/c/o/admin/panel/overview">
+  <button mat-button [matMenuTriggerFor]="homeMenu">
     <mat-icon svgIcon="home"></mat-icon><span matLine></span>
   </button>
+  <mat-menu #homeMenu="matMenu" class="mat-elevation-z6">
```
Comment (Mussuranae) [inline libs/admin/src/lib/admin/toolbar-top/toolbar-top.component.html:6]: You really need an elevation-6 ??? Oo'

File: libs/notification/src/lib/invitation/+state/invitation.model.ts [hunk 6]
```diff
@@ -1,4 +1,17 @@
...
+import { Organization } from '@blockframes/organization/+state/organization.model';
+import { PublicUser } from '@blockframes/user/+state/user.model';
+import { Movie } from '@blockframes/movie/+state/movie.model';
+import { Event } from '@blockframes/event/+state/event.model';
+
 export { InvitationStatus, createInvitation } from './invitation.firestore';
 
 export type Invitation = InvitationBase<Date>;
+
+export interface InvitationDetailed extends Invitation {
```
Comment (Mussuranae) [inline libs/notification/src/lib/invitation/+state/invitation.model.ts:11]: InvitationConfig ? InvitationEnhanced ? InvitationSettings ?

=> reference to the old comment `// @TODO (#2952) find better name and location` :D

File: libs/admin/src/lib/admin/toolbar-top/toolbar-top.component.html (hunk 5 above)
Comment (phpgeekfr) [reply libs/admin/src/lib/admin/toolbar-top/toolbar-top.component.html:6]: I don't even know what it is ^^ Will try without

**Pull request merged**